frontend/src/app/content/[id]/page.tsx

```
@@ -422,164 +422,159 @@ export default function Page() {
-              )}
+              <div className='spliter'></div>
 
+              {/* RIGHT SIDE - SUMMARY */}
+              <div className='content-summary'>
+                <p>Article Summary</p>
+              </div>
             </div>
+
+            {/* Render the Main Content */}
+            {formatedContent && (
+              <div dangerouslySetInnerHTML={{ __html: formatedContent }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)